runbooks/KubevirtHyperconvergedClusterOperatorCRModification.md

```diff
@@ -0,0 +1,30 @@
+# KubevirtHyperconvergedClusterOperatorCRModification
+
+## Meaning
+
+Hyperconverged Cluster Operator configures clusters in an opinionated way and overwrites its operands when there is an unexpected change to them. Users are expected to not modify the operands directly. The `HyperConverged` object is the source of truth for the configuration.
+
+This alert fires when an operand of Hyperconverged Cluster Operator is changed by someone else.
+
+
+## Impact
+
+When the operands are changed by someone else constantly, it may lead to oscillation in the cluster configuration and instabilities.
+
+
+## Diagnosis
+
+Check the alert details. `component_name` contains the type and name of the operand that is overwritten.
+
+```
+Labels
+    alertname=KubevirtHyperconvergedClusterOperatorCRModification
+    component_name=kubevirt-kubevirt-hyperconverged
+    severity=warning
+```
+In the example above, `kubevirt/kubevirt-hyperconverged` object is the operand for which the alert is triggered.
```

Comment:
to be more accurate we can say that the resource name itself contains the type, due to the fact that all kubevirt-hyperconverged CRs follow this pattern in their names: `<type>-kubevirt-hyperconverged`.

thus, if we use resource_type/resource_name notation, the example is:
`kubevirt.kubevirt.io/kubevirt-kubevirt-hyperconverged`

Reply:
@orenc1  Just updated. Could you please check it?

Reply:
yes, thanks.
lgtm on my side